.circleci/config.yml

```diff
@@ -99,7 +103,25 @@ workflows:
             - integration-tests
             - regression-integration-tests
           context: sonarcloud
+
 jobs:
+  skip_check:
+    docker: # run the steps with Docker
+      - image: cimg/openjdk:<< pipeline.parameters.java-tag >>
+    steps:
+      - checkout
+      - run:
+          name: evaluate [skipTests] condition
+          command: |
+            echo "export COMMIT_MESSAGE=\"$(git log --format=oneline -n 1 $CIRCLE_SHA1)\"" >> $BASH_ENV
+            source $BASH_ENV
+            echo "Commit message: $COMMIT_MESSAGE"
+            if [[ $COMMIT_MESSAGE =~ ^.*\[skipTests\].*$ ]]; then
```

Comment:
There's a [built in environment variable](https://circleci.com/docs/variables/#built-in-environment-variables) called `CIRCLE_BRANCH` that contains the name of the git branch. Could add a check that ignores the skipTests condition if the branch is develop